### PR TITLE
1448 transactions resolve trigger values before test execution

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds the ability for users to use environment variables during the test creation process.
It wires up the selected environment from the FE to the test creation and execution.

## Changes

- Resolve function for triggers
- Sending environment id when running tests from the FE.

## Fixes

- #1448 

## Checklist

- [x] tested locally
- [] added new dependencies
- [ ] updated the docs
- [x] added a test

https://www.loom.com/share/ec9a6b9728fb4cad93536a718539a6f1

### CODE REVIEW

File: server/executor/trigger/grpc.go [hunk 1]
```diff
@@ -104,6 +104,44 @@ func (t *grpcTriggerer) Type() model.TriggerType {
...
 }
 
+func (t *grpcTriggerer) Resolve(ctx context.Context, test model.Test, opts *TriggerOptions) (model.Test, error) {
+	grpc := test.ServiceUnderTest.GRPC
+
+	if grpc == nil {
+		return test, fmt.Errorf("no settings provided for GRPC triggerer")
+	}
+
+	address, err := opts.Executor.ResolveStatement(fmt.Sprintf("\"%s\"", grpc.Address))
```
Comment (mathnogueira) [inline server/executor/trigger/grpc.go:114]: isn't it worth creating a wrapInQuotes function?

File: server/http/mappings/http.go [hunk 2]
```diff
@@ -113,6 +115,8 @@ func (m Model) HTTPResponse(in openapi.HttpResponse) *model.HTTPResponse {
 
 func (m Model) Auth(in openapi.HttpAuth) *model.HTTPAuthenticator {
 	var props map[string]string
+	log.Println("@@@@ auth type", in.Type, in)
```
Comment (jorgeepc) [inline server/http/mappings/http.go:118]: 👀

File: web/src/components/EditTest/EditTest.tsx [hunk 3]
```diff
@@ -26,6 +26,7 @@ const EditTest = ({test}: IProps) => {
 
   const handleOnSubmit = useCallback(
     async (values: TDraftTest) => {
+      console.log('@@edit values', values);
```
Comment (jorgeepc) [inline web/src/components/EditTest/EditTest.tsx:29]: 👀

**Review by jorgeepc — APPROVED** [review]
Looks great my friend!

**Pull request merged**